server/src/main/kotlin/pishello/hello/persistence/database/ports/CardPort.kt

```diff
@@ -43,4 +43,17 @@ class CardPort(val repository: CardRepository, val userRepository: UserRepositor
         card.path = path
         return repository.save(card)
     }
+
+    fun updateData(id: Int, company: String?, name: String?, phone: String?, email: String?): Card? {
```

Comment:
sadly, there is no way change mode or add category from application now, I would add it